### PR TITLE
Return golint-compliant naming for mappings

### PR DESCRIPTION
**Pull request by LK4D4**
It was lost in pull request about runtime config

### CODE REVIEW

File: config_linux.go [hunk 1]
```diff
@@ -20,10 +20,10 @@ type Linux struct {
 // User specifies linux specific user and group information for the container's
 // main process.
 type User struct {
-	// Uid is the user id.
-	Uid int32 `json:"uid"`
+	// UID is the user id.
+	UID int32 `json:"uid"`
 	// Gid is the group id.
```
Comment (mrunalp) [inline config_linux.go]: Update comment.

File: runtime_config_linux.go [hunk 2]
```diff
@@ -11,9 +11,9 @@ type LinuxRuntimeSpec struct {
 
 type LinuxRuntime struct {
 	// UidMapping specifies user mappings for supporting user namespaces on linux.
```
Comment (mrunalp) [inline runtime_config_linux.go]: Here as well.

File: runtime_config_linux.go [hunk 3]
```diff
@@ -11,9 +11,9 @@ type LinuxRuntimeSpec struct {
...
-	UidMappings []IDMapping `json:"uidMappings"`
+	UIDMappings []IDMapping `json:"uidMappings"`
 	// UidMapping specifies group mappings for supporting user namespaces on linux.
```
Comment (mrunalp) [inline runtime_config_linux.go]: And here.

Comment (wking) [inline runtime_config_linux.go]: On Mon, Aug 31, 2015 at 04:11:40PM -0700, Mrunal Patel wrote:
> >  	// UidMapping specifies group mappings for supporting user namespaces on linux.
> 
> And here.

And UID → GID here too ;).

**Comment by LK4D4** [comment]
@mrunalp @wking Fixed. Thanks!

**Comment by laijs** [comment]
LGTM

**Comment by wking** [comment]
With the tip at 31485fa, this looks good to me.

**Comment by mrunalp** [comment]
Merging...

**Pull request merged**